examples/express-mysql/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@solarwinds-apm/example-express-mysql",
   "private": true,
   "scripts": {
-    "start": "node --loader solarwinds-apm/loader index.js"
+    "start": "node --import solarwinds-apm index.js"
```

Comment:
Examples run on Node 20 now and the `--loader` flag is deprecated for that version